### PR TITLE
Add qpid_proton dependency to Katello

### PR DESCRIPTION
**Pull request by jturel**
Needs https://github.com/theforeman/foreman-packaging/pull/6197

### CODE REVIEW

**Comment by jturel** [comment]
[test rpm]

**Comment by jturel** [comment]
@ehelms repoclosure is failing because tfm-rubygem(qpid_messaging) is missing. Did I omit something or is that expected at this point (or maybe something hasn't happened in a nightly job yet?)

**Comment by ehelms** [comment]
[test rpm]

**Comment by ehelms** [comment]
The nightly pipelin eneeded to run mash.

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 1]
```diff
@@ -78,6 +81,9 @@ BuildRequires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.14.1
 BuildRequires: %{?scl_prefix}rubygem(foreman_remote_execution) >= 3.0
 BuildRequires: %{?scl_prefix}rubygem(dynflow) >= 1.2.0
 BuildRequires: %{?scl_prefix}rubygem(activerecord-import)
+%if 0%{?rhel} == 7
+BuildRequires: %{?scl_prefix}rubygem(qpid_messaging)
+%endif
```
Comment (ehelms) [inline packages/katello/rubygem-katello/rubygem-katello.spec:86]: Given that this is in the auto-generated section but has a conditional around it, I think you should move this and the Requires to https://github.com/theforeman/foreman-packaging/pull/6200/files#diff-ea1969cc98962976d79819fceee9455b08e0f99c44e5071fa281f3718bb8e3f8R27

Comment (ehelms) [reply packages/katello/rubygem-katello/rubygem-katello.spec:86]: Related, see https://github.com/Katello/katello/pull/9088/files#r561227938

Comment (jturel) [reply packages/katello/rubygem-katello/rubygem-katello.spec:86]: Good call!

Comment (jturel) [reply packages/katello/rubygem-katello/rubygem-katello.spec]: @ehelms updated!

**Comment by jturel** [comment]
@ehelms this should be green once mashed? rpm builds are OK :)

**Comment by ehelms** [comment]
> @ehelms this should be green once mashed? rpm builds are OK :)

Correct. Which will happen later today when the katello pipeline runs.

**Comment by jturel** [comment]
[test rpm]

**Comment by jturel** [comment]
[test rpm]

**Comment by ehelms** [comment]
This will need https://github.com/theforeman/foreman-packaging/pull/6262

**Comment by jturel** [comment]
[test rpm]

**Comment by jturel** [comment]
@ehelms good to merge?

**Review by ehelms — APPROVED** [review]

**Pull request merged**